torchaudio/datasets/cmuarctic.py

```diff
@@ -90,7 +96,7 @@ def __init__(self,
                  root: str,
                  url: str = URL,
                  folder_in_archive: str = FOLDER_IN_ARCHIVE,
-                 download: bool = False) -> None:
```

Comment:
Interesting: why remove the typing annotation?

Reply:
This adds `-> None` to documentation, and I do not think it adds any value.

Reply:
We actually had an issue recently in PyTorch to add these "-> None" typing annotations to our inits. Check out:

https://github.com/pytorch/pytorch/issues/45497

You might save yourself a similar issue by leaving them in.

Reply:
oh good to know. Okay I will revert it.